src/app/(auth)/sign-in/page.tsx

```
@@ -1,5 +1,17 @@
+import { headers } from "next/headers";
+import { redirect } from "next/navigation";
+
 import { SignInView } from "@/modules/auth/ui/views/sign-in-view";
 
-export default function SignInPage() {
+export default async function SignInPage() {
+  const session = await auth.api.getSession({
+    headers: await headers(),
+  });
+
+  if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)